kobo/apps/openrosa/libs/utils/logger_tools.py

```diff
@@ -601,11 +607,6 @@ def safe_create_instance(
         )
     except ExpatError as e:
         error = OpenRosaResponseBadRequest(t("Improperly formatted XML."))
-    except ConflictingXMLHashInstanceError:
-        response = OpenRosaResponse(t('Conflict with already existing instance'))
-        response.status_code = 409
-        response['Location'] = request.build_absolute_uri(request.path)
-        error = response
     except DuplicateInstanceError:
         response = OpenRosaResponse(t('Duplicate instance'))
         response.status_code = 202
```

Comment:
We should return a 409 when it's a duplicate. 
Let confirm that with the rest of the team.

Reply:
Internal discussion: https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/Duplicated.20submission.20UUIDs/near/445782

Reply:
Will wait for John's comment